master/buildbot/changes/gitpoller.py

```diff
@@ -159,8 +170,16 @@ def poll(self):
             '+%s:%s' % (self._removeHeads(branch), self._trackerBranch(branch))
             for branch in branches
         ]
-        yield self._dovccmd('fetch',
```

Comment:
For me this test logged:

```
2015-03-22 12:23:57-0400 [-] --> buildbot.test.unit.test_changes_gitpoller.TestGitPoller.test_poll_GitError_loop <-- 
2015-03-22 12:23:57-0400 [-] command init ['--bare', 'gitpoller-work'] in None on repourl git@example.com:foo/baz.git failed with exit code 128:⋅
```
with no attending exception, so that looks great